src/vcpkg/configure-environment.cpp

```diff
@@ -207,12 +251,55 @@ namespace vcpkg
         return result;
     }
 
-    int run_configure_environment_command(const VcpkgPaths& paths, StringView arg0, View<std::string> args)
+    bool more_than_one_mapped(View<const StringLiteral*> candidates, const std::set<std::string, std::less<>>& switches)
```

Comment:
```suggestion
    static bool more_than_one_mapped(View<const StringLiteral*> candidates, const std::set<std::string, std::less<>>& switches)
```

I think this function is only used in this file now.